### PR TITLE
Save identity and add eslint config

### PR DESCRIPTION
**Pull request by Aitorpm**

### CODE REVIEW

File: lib/accessToken.js [hunk 1]
```diff
@@ -51,27 +51,28 @@ function accessToken({appName, authToken, serviceChatSid, serviceVoiceSid,
...
     // });
-    res.set('Content-Type', 'application/json');
+    res.set('Content-Type', 'application/json')
     res.send(token.toJwt())
   }
 
   function voice(req, res, next)
   {
     const {deviceId, identity} = req.body
+    process.env.identity = identity
```
Comment (piranna) [inline lib/accessToken.js]: Why are you setting this on `process.env`?

Comment (Aitorpm) [reply lib/accessToken.js]: Because we need to save the identity and use then in the incoming calls client.

Comment (piranna) [reply lib/accessToken.js]: If you need to have set it globally use the `global` object for that, but it's an anitpattern in Node.js.

Comment (Aitorpm) [reply lib/accessToken.js]: Yes, I know it but is necessary save this data to receive incoming calls. Probably there is a different method that couldn't be necessary.

**Pull request merged**